src/main/java/com/arcbees/website/client/application/products/chosen/ChosenView.java

```diff
@@ -16,18 +16,36 @@
 
 package com.arcbees.website.client.application.products.chosen;
 
+import com.arcbees.chosen.client.Chosen;
+import com.google.gwt.dom.client.SelectElement;
+import com.google.gwt.event.logical.shared.AttachEvent;
 import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
 import com.google.gwt.user.client.ui.Widget;
 import com.google.inject.Inject;
 import com.gwtplatform.mvp.client.ViewImpl;
 
-public class ChosenView extends ViewImpl implements ChosenPresenter.MyView {
+import static com.google.gwt.query.client.GQuery.$;
+
+public class ChosenView extends ViewImpl implements ChosenPresenter.MyView, AttachEvent.Handler {
     interface Binder extends UiBinder<Widget, ChosenView> {
     }
 
+    @UiField
+    SelectElement selectToEnhance;
+
     @Inject
     ChosenView(
             Binder binder) {
         initWidget(binder.createAndBindUi(this));
+
+        asWidget().addAttachHandler(this);
```

Comment:
Using latest GWTP, you can remove this and use onAttach and onDetach overridable methods.